_includes/tutorials/console-consumer-primitive-keys-values/kafka/code/tutorial-steps/dev/expected-output-step-two.txt

```diff
@@ -8,3 +8,4 @@
 4 : -80.832911
 4 : -2.2259418
 7 : 30.838015
+Processed a total of 10 messages
```

Comment:
This is a side-fix that was causing another test to fail.  I guess it got dropped somehow from one of the rebases.